tox.ini

```diff
@@ -12,13 +11,13 @@ requires =
 pypi_filter = https://raw.githubusercontent.com/astropy/ci-helpers/master/pip_pinnings.txt
 commands =
     pip freeze
-    python setup.py test
+    pytest --pyargs pyvo
 
 description =
     run tests
-    astropy30: with astropy 3.0.*
-    astropy31: with numpy 3.1.*
-    astropy32: with numpy 3.2.*
+    astropy40: with astropy 4.0.*
```

Comment:
You might also want to match your `astropy` minversion to your `numpy` minversion for cross-testing.

Reply:
Sorry, I don't understand. We don't have min versions specified and we've decided not to do cross-testing here, no?

Reply:
In that case, I still recommend cross testing with the minversion of numpy that astropy 4.0 supports (numpy 1.16), but if you disagree, feel free to ignore; this is just a recommendation.